electron/process_manager.ts

```diff
@@ -237,27 +213,24 @@ function checkUdpForwardingEnabled() {
         },
         (err, socket, info) => {
           if (err) {
-            console.error(`Failed to create UDP connection to local proxy: ${err.message}`);
-            reject(new Error());
+            reject(new errors.RemoteUdpForwardingDisabled(`could not connect to local proxy`));
             return;
           }
           const dnsRequest = getDnsRequest();
           const packet = socks.createUDPFrame({host: '1.1.1.1', port: 53}, dnsRequest);
           const udpSocket = dgram.createSocket('udp4');
 
-          udpSocket.on('error', (err) => {
-            const msg = `UDP socket failure: ${err}`;
-            console.error(msg);
-            reject(new Error(msg));
+          udpSocket.on('error', (e) => {
+            reject(new errors.RemoteUdpForwardingDisabled('UDP socket failure'));
           });
 
           udpSocket.on('message', (msg, info) => {
-            console.info('UDP forwarding enabled');
             stopUdp();
             resolve();
           });
 
           // Retry sending the query every second.
+          // TODO: logging here is a bit verbose
```

Comment:
Should we remove/fix the unnecessary  logging to avoid the TODO?

Reply:
I didn't do that because I could imagine these log lines being useful during debugging. Without doing some testing it wasn't obvious what was expected/unexpected here - if you think it's safe to remove them, I'm happy to :-)